embeddings/data/data_collator.py

```diff
@@ -0,0 +1,45 @@
+from dataclasses import dataclass
+from typing import Any, Dict, List, Union
+
+import torch
+from transformers import BatchEncoding, DataCollatorForTokenClassification
+
+
+# ignoring `Class cannot subclass "DataCollatorForTokenClassification" (has type "Any")` error
+@dataclass
+class CustomDataCollatorForTokenClassification(DataCollatorForTokenClassification):  # type: ignore
```

Comment:
where is a difference in the logic of the custom one? I have read the original version and i do not see it https://github.com/huggingface/transformers/blob/19e5ed736611227b004c6f55679ce3536db3c28d/src/transformers/data/data_collator.py#L263

Reply:
i removed the labels from dictionary that is passed to tokenizer for padding because it does not pad the labels and when the tokenizer returns BatchEncoding object it tries to convert the whole dict to tensor and it fails